src/LiveComponent/src/Twig/LiveComponentRuntime.php

```diff
@@ -45,4 +47,25 @@ public function getComponentUrl(string $name, array $props = []): string
 
         return $this->urlGenerator->generate($metadata->get('route'), $params, $metadata->get('url_reference_type'));
     }
+
+    public function getLiveAction(string $actionName, array $parameters = [], array $modifiers = [], ?string $event = null): string
+    {
+        $attributes = $this->stimulusHelper->createStimulusAttributes();
+
+        $prevent = false;
+        if (\array_key_exists('prevent', $modifiers) && $modifiers['prevent']) {
+            $prevent = true;
+            unset($modifiers['prevent']);
+        }
```

Comment:
If we keep this implementation, you should unset the modifier even if it is false or null or this will generate an modifier the JS live controller won't be able to parse anyway